interp/notation.ml

```diff
@@ -408,6 +408,8 @@ type target_kind =
 type string_target_kind =
   | ListByte
   | Byte
+  | PString
+  | IntCharCode
```

Comment:
I'm not sure that's possible since `int` is not an inductive (at least I could not make it work).

Reply:
Even using `int_wrap`? (or something like that, take inspiration of sint63.v for instance)

Reply:
Ah, I managed to make this work. The issue I had was that the string notation stuff had to be registered first...

One limitation of doing what you suggest, though, is that if we want to support writing `"255"%char63` then we need to implement the parsing in Coq instead of OCaml (currently I'm using the same code as for the `Byte` type, so this just works). I think I would prefer keeping the current code in the end, but that's up to you: what do you think?

Reply:
Well, if really needed this can still be implemented in gallina, no need for the OCaml code. But isn't `255%uint63` enough?

Reply:
Yeah, that's indeed probably enough.